agito.py

```diff
@@ -39,6 +39,10 @@
 MERGEINFO_LINE_RE = re.compile(r'([\w/\-]+):.*-(\d+)')
 WORD_MATCH = re.compile(r'(\s*)(\S+)')
 BULLET_POINT_RE = re.compile(r'(\s*[\*\+]\s*)')
+COMMIT_REF_REs = (
+	re.compile(r"r([0-9.]+)"),
```

Comment:
yes, if used in seldom, here it means dot in char range, and it is so in any regexp engine i've encountered so far. the cases where it was escaped, it was just illusion that one thought, oh, i escape dot, but `[\.]` just means backslash or dot and author never encountered the mismatch because he just never tested.